CMakeLists.txt

```
@@ -163,6 +163,11 @@ if(WIN32)
 	# so that build environments which lack a C compiler, but have a C++
 	# compiler may build ninja.
 	set_source_files_properties(src/getopt.c PROPERTIES LANGUAGE CXX)
+
+	# windows.h defines min() and max() which conflict with std::min()
+	# and std::max(), which both might be used in sources. Avoid compile
+	# errors by telling windows.h to not define those two.
+	add_definitions(-DNOMINMAX)
```

```suggestion
	add_compile_definitions(NOMINMAX)
```
I think `add_compile_definitions` is preferred nowadays and it would be consistent to the rest of the file.

---

done